addons/l10n_it_edi/models/account_move.py

```diff
@@ -1341,7 +1341,7 @@ def _l10n_it_edi_send(self, attachments_vals):
             for filename in filename_move:
```

Comment:
Again, not for this particular PR
@odoo/rd-accounting the translated messages of this file would benefit from a grammar review :grimacing:

Reply:
Ok, I already got some fixes to do on the localization in `master`, can also do grammar check.
Draft: https://github.com/odoo/odoo/pull/165673